Automattic/themes
# Varia: Fix cover text color over clear backgrounds.

#### Changes proposed in this Pull Request:
 When you add a cover block with a white (or clear) background the font color is also white and you cannot see what you write. 

This change makes the necessary changes so that the cover font colors are picked up from the customizer palette in `Colors & Backgrounds > Text Color`. 

This fix applies to Varia and all its child themes.

| Before | After |
|-------|-------|
|![image](https://user-images.githubusercontent.com/375980/122973327-7f8d7000-d367-11eb-82b3-4deea83eabb7.png)|![image](https://user-images.githubusercontent.com/375980/122973628-d2672780-d367-11eb-8eb4-4d31e69826bf.png)|

The changes are in the sass files in the Varia template:
* `varia/sass/blocks/utilities/_style.scss`
* `varia/sass/blocks/utilities/_editor.scss`
* `varia/sass/blocks/cover/_style.scss`
* `varia/sass/blocks/cover/_editor.scss`

All the other changes are auto-generated following this instructions: pNEWy-daO-p2

**This also fixes the alignment of the search block on the Rivington theme**. In this commit:  https://github.com/Automattic/themes/commit/87dbce487c959e4d701234be220777247fcfe350 changes where added directly to the css files instead of the corresponding sass file. When I rebuilt Varia children the changed got removed.

SVN: D63216-code

#### Testing instructions
 1. Clone this repo `git@github.com:Automattic/themes.git`
 2. Push this repo to your sandbox using `./sandbox.sh push` (more instructions in README.md https://github.com/Automattic/themes)
 3. Verify on your sandbox that when you add a cover block with a light background-color the paragraph inside is visible. This only applies to Varia and its child themes. Here is a list of the themes affected:
```
Barnsbury
Dalston
Mayland
Rivington
Maywood
Balasana
Shawburn
Alves
Exford
Rockfield
Stratford
Coutoire
Morden
Stow
Hever
Leven
Varia
```
 
#### Related issue(s):
https://github.com/Automattic/themes/issues/3483
